driver/bn_driver.hpp

```diff
@@ -1443,7 +1443,7 @@ int BatchNormDriver<Tgpu, Tref, Tmix>::VerifyBackward()
     dbias_dev->FromGPU(GetStream(), dbias.data());
 #if(MIO_BN_DEBUG == 1)
     const Tref tolerance =
-        static_cast<Tref>(1000 * ((sizeof(Tgpu) == 4) ? ERRTOL_FP32 : ERRTOL_FP16));
+        static_cast<Tref>(1000 * (sizeof(Tgpu) == 4) ? ERRTOL_FP32 : ERRTOL_FP16);
```

Comment:
This part should be retained.

Reply:
Thanks! This is urgent hotfix for Release builds, that do not normally set MIO_BN_DEBUG, I guess, so there should be no big harm. This change will be back when #1047 is resolved. Is it acceptable for you?